test/src/test/java/org/corfudb/infrastructure/log/BoundedMapTest.java

```diff
@@ -63,7 +63,7 @@ public void testConstructorConditions() {
         assertThatThrownBy(() -> new BoundedMap(Long.MAX_VALUE, 1))
                 .isInstanceOf(ArithmeticException.class);
 
-        assertThatThrownBy(() -> new BoundedMap(Long.MAX_VALUE - 10, 5))
```

Comment:
Nice catch.

So the condition was failing and the test was passing, but not for the right reasons.